packages/now-cli/src/commands/dev/dev.ts

```diff
@@ -50,21 +51,26 @@ export default async function dev(
     return 1;
   }
 
-  let devCommand: undefined | string;
-  let frameworkSlug: null | string = null;
+  let devCommand: string | undefined;
+  let frameworkSlug: string | undefined;
+  let projectSettings: ProjectSettings | undefined;
   if (link.status === 'linked') {
     const { project, org } = link;
     client.currentTeam = org.type === 'team' ? org.id : undefined;
 
+    projectSettings = project;
+
     if (project.devCommand) {
       devCommand = project.devCommand;
     } else if (project.framework) {
       const framework = frameworks.find(f => f.slug === project.framework);
 
       if (framework) {
-        frameworkSlug = framework.slug;
-        const defaults = framework.settings.devCommand;
+        if (framework.slug) {
+          frameworkSlug = framework.slug;
+        }
```

Comment:
I'm curious, why add the `if` statement here?

Assigning `frameworkSlug = framework.slug` every time was causing an issue?

Reply:
I see, its to convert null to undefined, seems fine 👍